components/HowToPanel.tsx

```diff
@@ -88,13 +98,16 @@ const HowToPanel = (props) => {
                 <input
                   id='copy-code'
                   className='input-group-element'
-                  readOnly='readOnly'
+                  readOnly={true}
```

Comment:
you don't need `={true}`, only `readOnly` works the same. Also, it's cleaner to read.